js-intro.md

```diff
@@ -30,6 +41,7 @@ Answer the following questions. First, without external resources. Challenge you
 4. In regards to functions, what is an argument? What is the difference between an argument and a variable?
 
   Your answer:
+  Argument is a object that represents a function to 
 
```

Comment:
An argument is a placeholder value that is passed into a function. A variable is a "bucket" for code that is declared using var, let or const and assigned using a single =